### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by AdamLalor**

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@AdamLalor There is some extra change included here other than the README. something relating to Semantic i think. If you want to include this can you do that in anther pr?

**Comment by laurafitzgerald** [comment]
@AdamLalor there is probably some more info that you can include here. Here are a few samples that you could take sections from: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2 There's a bunch of samples in here: https://github.com/matiassingers/awesome-readme

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # SchoolWebsite
-test
-K.H
\ No newline at end of file
+
+## This is a school website that is being made by 4 Mount Sion pupils with the help of RedHat
```
Comment (ciaranRoche) [inline README.md]: I would suggest to remove the `##` as this setting the text as header when we want this to be a paragraph of regular text, also I would suggest to rewording this paragraph also

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # SchoolWebsite
```
Comment (ciaranRoche) [inline README.md]: A more descriptive title, something like `Mount Sion CBS Secondary School Website`

**Comment by ciaranRoche** [comment]
Closing this as these changes will be made https://github.com/MountSionCBSSecondary/SchoolWebsite/pull/13 PR

**Pull request closed without merging**